code/modules/research/stock_parts.dm

```diff
@@ -273,6 +283,48 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	rating = 4
 	materials = list(/datum/material/iron=80)
 
+//Rating 5
+
+/obj/item/stock_parts/capacitor/cubic
+	name = "cubic capacitor"
+	desc = "A cubic capacitor used in the construction of a variety of devices."
+	icon_state = "cubic_capacitor"
+	rating = 5
+	materials = list(/datum/material/iron=50, /datum/material/glass=50)
+	resistance_flags = INDESTRUCTIBLE
+
+/obj/item/stock_parts/scanning_module/hexaphasic
+	name = "hexaphasic scanning module"
+	desc = "A compact, ultra resolution hexaphasic scanning module used in the construction of certain devices."
+	icon_state = "hexaphasic_scan_module"
+	rating = 5
+	materials = list(/datum/material/iron=50, /datum/material/glass=20)
+	resistance_flags = INDESTRUCTIBLE
+
+/obj/item/stock_parts/manipulator/planck
+	name = "\improper Planck-manipulator"
+	desc = "A tiny manipulator used in the construction of certain devices."
+	icon_state = "planck_mani"
+	rating = 5
+	materials = list(/datum/material/iron=30)
+	resistance_flags = INDESTRUCTIBLE
+
+/obj/item/stock_parts/micro_laser/quinthyper
+	name = "quint-hyper micro-laser"
```

Comment:
Could be better as "Quintic-Hyper Micro-Laser" rather than Quint-hyper

Reply:
why wouldn't it be named atto as SI prefixes are currently being used. Micro>Nano>Pico>Femto>Atto>Zepto(if tier 6 one day)

https://en.wikipedia.org/wiki/Metric_prefix

Reply:
It follows the same format as "quad-ultra micro-laser"

Reply:
this name is stupid